pkg/apis/operator/v1alpha1/tektonaddon_validation_test.go

```diff
@@ -94,35 +94,5 @@ func Test_ValidateTektonAddon_InvalidParamValue(t *testing.T) {
 	}
 
 	err := ta.Validate(context.TODO())
-	assert.Equal(t, "invalid value: test: spec.params.clusterTasks[0]", err.Error())
-}
-
-func Test_ValidateTektonAddon_ClusterTaskIsFalseAndPipelineTemplateIsTrue(t *testing.T) {
```

Comment:
Do we have test checking the same for resolver tasks?

Reply:
we need to add the tests, I add missing tests